frontend/common/src/components/UserProfile/ExperienceByTypeListing.tsx

```diff
@@ -96,41 +96,51 @@ const ExperienceByTypeListing: React.FunctionComponent<
 
   return (
     <>
-      <ExperienceByType
-        title={intl.formatMessage({ defaultMessage: "Personal" })}
-        icon={<LightBulbIcon style={{ width: "1.5rem" }} />}
-        experiences={personalExperiences}
-        defaultOpen={defaultOpen}
-        experienceEditPaths={editPaths}
-      />
-      <ExperienceByType
-        title={intl.formatMessage({ defaultMessage: "Community" })}
-        icon={<UserGroupIcon style={{ width: "1.5rem" }} />}
-        experiences={communityExperiences}
-        defaultOpen={defaultOpen}
-        experienceEditPaths={editPaths}
-      />
-      <ExperienceByType
-        title={intl.formatMessage({ defaultMessage: "Work" })}
-        icon={<BriefcaseIcon style={{ width: "1.5rem" }} />}
-        experiences={workExperiences}
-        defaultOpen={defaultOpen}
-        experienceEditPaths={editPaths}
-      />
-      <ExperienceByType
-        title={intl.formatMessage({ defaultMessage: "Education" })}
-        icon={<BookOpenIcon style={{ width: "1.5rem" }} />}
-        experiences={educationExperiences}
-        defaultOpen={defaultOpen}
-        experienceEditPaths={editPaths}
-      />
-      <ExperienceByType
-        title={intl.formatMessage({ defaultMessage: "Award" })}
-        icon={<StarIcon style={{ width: "1.5rem" }} />}
-        experiences={awardExperiences}
-        defaultOpen={defaultOpen}
-        experienceEditPaths={editPaths}
-      />
+      {personalExperiences.length > 0 && (
+        <ExperienceByType
+          title={intl.formatMessage({ defaultMessage: "Personal" })}
+          icon={<LightBulbIcon style={{ width: "1.5rem" }} />}
+          experiences={personalExperiences}
+          defaultOpen={defaultOpen}
+          experienceEditPaths={editPaths}
+        />
+      )}
```

Comment:
When checking length, it's safer to use the following pattern as this may output `0` or `false` to the screen.

```tsx
{array.length ? (<Component />) : null}
```

Reply:
Changed in [fb6fcdd](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3452/commits/fb6fcdd3092b26acce1e1e50bab554c3e73ee333).